### PR TITLE
Separated class build from define

### PR DESCRIPTION
**Pull request by georges-gomes**
Hi!
https://webcomponents.dev works better with the class than direct define.
With this we can release a support for Riotjs 👍
Thanks

### CODE REVIEW

**Comment by cekvenich** [comment]
Cool thing about standard custom elements is that they work on ie11. Would this facilitate IE11 support?

**Comment by georges-gomes** [comment]
@cekvenich I don't think it make any difference because the polyfill is probably at the `customElement.define()` level.

**Comment by cekvenich** [comment]
Yes, I can pre-load anything needed before it gets to that. I look forward to a regular example. I will then try to make that example ie11.

File: index.next.js [hunk 1]
```diff
@@ -26,22 +26,20 @@ function moveChildren(source, target) {
...
- * Create a new custom element using the riot core components
- * @param   {string} name - custom component tag name
+ * Create a new custom element Class using the riot core components
  * @param   {Object} api - custom component api containing lifecycle methods and properties
- * @param   {Object} options - optional options that could be passed to customElements.define
- * @returns {undefined} it's a void method again ¯\_(ツ)_/¯
+ * @returns {Class} Class extends HTMLElement
  */
-export default function define(name, api, options) {
+export function buildElementClass(api) {
```
Comment (GianlucaGuarini) [inline index.next.js]: could you please rename it `createElementClass` instead? thank you

Comment (georges-gomes) [reply index.next.js]: Makes more sense indeed 👍

**Comment by GianlucaGuarini** [comment]
@cekvenich please make sure the test pass

**Comment by georges-gomes** [comment]
My fault, I'm sorry

**Comment by GianlucaGuarini** [comment]
thank you

**Comment by GianlucaGuarini** [comment]
this feature was added in v4.1.0

**Comment by georges-gomes** [comment]
👍It's working! We will release soon

**Comment by GianlucaGuarini** [comment]
@georges-gomes thank you for taking the time for it

**Pull request merged**